pysoa/test/stub_service.py

```diff
@@ -304,10 +308,18 @@ def __enter__(self):
 
         @wraps(Client.call_actions)
         def wrapped(client, service_name, actions, *args, **kwargs):
+            assert isinstance(service_name, six.text_type), 'Called service name "{}" must be unicode'.format(
+                service_name,
+            )
+
             requests_to_send_to_mock_client = OrderedDict()
             requests_to_send_to_wrapped_client = []
             for i, action_request in enumerate(actions):
                 request_action = getattr(action_request, 'action', None) or action_request['action']
+                assert isinstance(request_action, six.text_type), 'Called action name "{}" must be unicode'.format(
+                    request_action,
```

Comment:
`request_action` and `action_request` are confusing names. Could one of them be renamed?

Reply:
Done.